src/plugins/device_license.c

```diff
@@ -11,14 +11,17 @@
 #include <openssl/err.h>
 #include <pthread.h>
 #include <stdio.h>
+#include <errno.h>
 #include <string.h>
 #include <sys/stat.h>
 #include <sys/time.h>
 
 #define SECONDS_PER_DAY 86400
 
-const char* DEVICE_LICENSE_FILE_PATH = "/var/lib/xr_driver/device_license";
-const char* DEVICE_LICENSE_TEMP_FILE_PATH = "/var/lib/xr_driver/device_license.tmp";
+const char* BACKUP_DIR = "/.local/state";
```

Comment:
A little more context in the naming would be nice, e.g. `XDG_FALLBACK_DIR` or something similar